layouts/404.html

```diff
@@ -0,0 +1,17 @@
+{{ define "main"}}
+<div class="contain error-page">
+  <h1>404 Page not found</h1>
+  <ul>
+    {{ $p := . }}
+    {{ range .Site.Menus.main }}
+    <li class="nav-item mr-2 mr-lg-4 mt-1 mt-lg-0">
+      {{ with .Page }}
+      {{ end }}
+      {{ $url := urls.Parse .URL }}
+      {{ $baseurl := urls.Parse $.Site.Params.Baseurl }}
+      <a href="{{ with .Page }}{{ .RelPermalink }}{{ else }}{{ .URL | relLangURL }}{{ end }}" {{ if ne $url.Host $baseurl.Host }}target="_blank" {{ end }}>{{ .Name }}</a>
```

Comment:
This would probably be cleaner if spread across multiple lines, like:
```html
  <a
    href="{{ with .Page }}{{ .RelPermalink }}{{ else }}{{ .URL | relLangURL }}{{ end }}"
    {{ if ne $url.Host $baseurl.Host }}target="_blank"{{ end }}
  >
    {{ .Name }}
  </a>
```